client/src/components/CreateBoard.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
```

Comment:
Thank you for creating this. 
Is this same structure as the one for  creating column?
As I mentioned before, if it is the same, we would be able to change this as reusable.
Since this component would be needed when user access dashboard, I will change this as reusable when I work on the task.